addons/l10n_hk/models/res_bank.py

```diff
@@ -41,9 +41,6 @@ def _get_merchant_account_info(self):
             return f'0012hk.com.hkicl{fps_account}'
         return super()._get_merchant_account_info()
 
-    def _get_additional_data_field(self, comment):
-        return f'05{len(comment):02}{comment}'
-
```

Comment:
Keep the diff simple, just return false here and add a comment.
I believe we will add it back in master but optional right? So no need to remove everything :)

Reply:
master already have this, therefore I disabled the forward-port.